src/Debug.h

```diff
@@ -185,14 +185,16 @@ class Raw
 {
 public:
     Raw(const char *label, const char *data, const size_t size):
-        level(-1), label_(label), data_(data), size_(size), useHex_(false) {}
+        level(-1), label_(label), data_(data), size_(size), useHex_(false), useGap_(true) {}
 
     /// limit data printing to at least the given debugging level
     Raw &minLevel(const int aLevel) { level = aLevel; return *this; }
 
     /// print data using two hex digits per byte (decoder: xxd -r -p)
     Raw &hex() { useHex_ = true; return *this; }
 
+    Raw &gap(bool useGap = true) { useGap_ = useGap; return *this; }
```

Comment:
Actually it is used inside HttpTunneler.cc
Without this new feature the Raw always prints a space before data if there is not any label. The gap() allow us to not print the space before raw data.
Looks minor but if you are debugging server answer an existing leading space may be confusing.

However If you insist I will remove this part.

Reply:
> This debugs gap() feature addition is not actually needed by this PR

@yadij, why do you think this _used_ feature is "not needed"? If you insist, it can be moved into a separate PR, but I see no good reason to insist on that -- it is a small, safe, localized, and seemingly non-controversial improvement used by the PR code. I very much doubt that clean commit isolation benefits are worth the overheads of a separate PR in this case. Do you insist on a separate PR?

Reply:
ah, I misread the default behaviour as being no-gap.